content/apps/managing-teammates.md

```diff
@@ -0,0 +1,24 @@
+---
+menu:
+  main:
+    parent: advanced
+title: Managing Teammates
+weight: 10
+---
+
+To add a new teammate to your organization:
+
+1. [Send them an invite](https://login.cloud.gov/invitations/new).
+1. Run
+
+    ```bash
+    cf set-org-role <email> <org> OrgManager
+    ```
+
+1. Add them to each space:
+
+    ```bash
+    cf set-space-role <email> <org> <space> SpaceDeveloper
```

Comment:
It shouldn't be `SpaceManager` unless you intend for that user to also be able to manage space roles for other people.

It's possible to set roles through the deck, yes, but the UX is kind of sucky right now so I wouldn't send them there. Or if you do let them know it may be easier but that we're looking for feedback on an early product. 

(You should probably add a note to make sure they're running the updated CLI, though, since they may get errors from the CLI otherwise and I'm sure very few people have updated to the version that eliminates them.0